src/test/java/com/commercetools/sync/commons/asserts/statistics/ProductTypeSyncStatisticsAssert.java

```diff
@@ -0,0 +1,13 @@
+package com.commercetools.sync.commons.asserts.statistics;
+
+import com.commercetools.sync.producttypes.helpers.ProductTypeSyncStatistics;
+
+import javax.annotation.Nullable;
+
+public class ProductTypeSyncStatisticsAssert extends
```

Comment:
Great that you are creating your own statistics assertion 🎉 

but maybe `final` 😊 ?

Reply:
23f9597c9a5e0d2d072c0622d84677bd8e034569